backend/core/views.py

```
@@ -2257,9 +2306,9 @@ def _import_objects(self, parsed_data: dict, domain_name: str):
 
         except ValidationError as e:
             if missing_libraries:
-                logger.warning(f"Missing libraries: {missing_libraries}")
+                logger.warning("Missing libraries", libraries=missing_libraries)
                 raise ValidationError({"missing_libraries": missing_libraries})
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)